solvebio/cli/data.py

```diff
@@ -534,6 +537,10 @@ def _download_recursive(full_path, local_folder_path, dry_run=False,
     for file_obj in results:
         depth = len(file_obj.path.split("/"))
         file_obj.depth = depth
+        # MD5 not retrieved by GlobalSearch so
```

Comment:
only look for md5 if the "object_type" is "file"

Reply:
Ah right. I realized that we couldn't do MD5 search by "if file > 1MB" because GlobalSearch doesn't return file size either. Will make this change.

Reply:
Ahh ok. If you want you can add a feature request ticket to jira for exposing md5 and file size via the base GlobalSearch record schema.

Reply:
but this is fine for now I think